app/vlselect/logsql/logsql.go

```diff
@@ -938,6 +938,55 @@ func ProcessQueryRequest(ctx context.Context, w http.ResponseWriter, r *http.Req
 	}
 }
 
+// ProcessAdminTenantsRequest processes /select/admin/tenants request.
+func ProcessAdminTenantsRequest(ctx context.Context, w http.ResponseWriter, r *http.Request) {
+	start, okStart, err := getTimeNsec(r, "start")
+	if err != nil {
+		httpserver.Errorf(w, r, "%s", err)
+		return
+	}
+	end, okEnd, err := getTimeNsec(r, "end")
+	if err != nil {
+		httpserver.Errorf(w, r, "%s", err)
+		return
+	}
+	if !okStart {
+		start = math.MinInt64
+	}
+	if !okEnd {
+		end = math.MaxInt64
+	}
+
+	sw := &syncWriter{
+		w: w,
+	}
+
+	var bwShards atomicutil.Slice[bufferedWriter]
+	bwShards.Init = func(shard *bufferedWriter) {
+		shard.sw = sw
+	}
+	defer func() {
+		shards := bwShards.All()
+		for _, shard := range shards {
+			shard.FlushIgnoreErrors()
+		}
+	}()
+
+	w.Header().Set("Content-Type", "application/json")
+
+	tenants, err := vlstorage.GetTenantIDs(ctx, start, end)
+	if err != nil {
+		httpserver.Errorf(w, r, "cannot obtain tenantIDs: %s", err)
+		return
+	}
+
+	_, err = fmt.Fprintf(w, "{\"status\":\"success\",\"data\":%q}\n", tenants)
+	if err != nil {
+		httpserver.Errorf(w, r, "cannot obtain tenantIDs: %s", err)
+		return
+	}
```

Comment:
Encoding `"data\":%q` data as quouted string causes it to be double escaped, for example:
```bash
curl localhost:9428/select/admin/tenants -d start=2022-07-13T12:16:51.229Z -d end=2025-07-12T12:16:51.229Z
{"status":"success","data":"[{\"accountID\":0,\"projectID\":0},{\"accountID\":10,\"projectID\":0},{\"accountID\":12,\"projectID\":11},{\"accountID\":10,\"projectID\":10},{\"accountID\":12,\"projectID\":1},{\"accountID\":12,\"projectID\":10}]"}
```
Parsed as JSON:
```json
{
  "status": "success",
  "data": "[{\"accountID\":12,\"projectID\":1},{\"accountID\":12,\"projectID\":10},{\"accountID\":12,\"projectID\":11},{\"accountID\":0,\"projectID\":0},{\"accountID\":10,\"projectID\":0},{\"accountID\":10,\"projectID\":10}]"
}
```

Reply:
Fixed
<img width="1218" alt="Screenshot 2025-05-09 at 14 39 40" src="https://github.com/user-attachments/assets/a790817f-1ff5-4804-b918-7c7d94175f3c" />